src/magnet.js

```diff
@@ -532,13 +536,20 @@ class Magnet {
    * Starts application.
    */
   async start() {
+    this.maybeRunLifecycleFile_(Magnet.LifecyleFiles.BEFORE_START);
+
+    this.setupMiddlewares_();
+    this.setupApplicationSettings_();
```

Comment:
Does moving `setupMiddlewares_` and `setupApplicationSettings_` to can potentially cause issues with existing users.

Reply:
I don't think so, express is not invoked on build function.

Pretty much build and start are the main functions. It was just manipulating express before the time it's used actually.